include/crm/common/logging.h

```diff
@@ -113,7 +102,7 @@ unsigned int get_crm_log_level(void);
 /*
  * Throughout the macros below, note the leading, pre-comma, space in the
  * various ' , ##args' occurrences to aid portability across versions of 'gcc'.
- *	http://gcc.gnu.org/onlinedocs/cpp/Variadic-Macros.html#Variadic-Macros
+ * https://gcc.gnu.org/onlinedocs/cpp/Variadic-Macros.html#Variadic-Macros
```

Comment:
Just a (possibly forward) reference:
One can make do without this GCCism just with plain C99, even though
sometimes it wlll become rather hefty.